docs/multicluster/setup-multi-primary.sh

```diff
@@ -17,11 +17,14 @@
 # This script sets up a primary-primary multi-network Istio mesh locally with kind using the Sail operator.
 # It is adapted from: https://istio.io/latest/docs/setup/install/multicluster/multi-primary_multi-network/.
 
+ISTIO_VERSION=${1:-"master"}
+
 while [ $# -gt 0 ]; do
   key="$1"
   case $key in
     -h|--help)
       echo "Usage: setup-multi-primary.sh <istio_version>"
```

Comment:
nit: s/`<istio_version>`/`[istio_version]`/ (it's optional)

maybe mention if not supplied, `master` is implied.